Makefile.kind

```diff
@@ -356,6 +358,33 @@ kind-servicemesh-install-cilium: check_deps kind-ready ## Install a local Cilium
 	@echo "Deploying L2-Announcement Policy..."
 	kubectl apply -f $(ROOT_DIR)/contrib/testing/servicemesh/l2policy.yaml
 
+.PHONY: kind-servicemesh-prereqs
+kind-servicemesh-prereqs: check_deps kind-ready
+	@echo "  SETUP Servicemesh"
+	kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/$(GW_VERSION)/config/crd/$(GW_CHANNEL)/gateway.networking.k8s.io_gatewayclasses.yaml
+	kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/$(GW_VERSION)/config/crd/$(GW_CHANNEL)/gateway.networking.k8s.io_gateways.yaml
+	kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/$(GW_VERSION)/config/crd/$(GW_CHANNEL)/gateway.networking.k8s.io_httproutes.yaml
+	kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/$(GW_VERSION)/config/crd/$(GW_CHANNEL)/gateway.networking.k8s.io_referencegrants.yaml
+	kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/$(GW_VERSION)/config/crd/$(GW_CHANNEL)/gateway.networking.k8s.io_grpcroutes.yaml
+	kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/$(GW_VERSION)/config/crd/experimental/gateway.networking.k8s.io_tlsroutes.yaml
```

Comment:
should this have s/experimental/$(GW_CHANNEL)/?

Reply:
TLSRoute is in experimental channel only now.
From
https://gateway-api.sigs.k8s.io/guides/tls/#tls-configuration
:
```
The TLSRoute and BackendTLSPolicy resources described below are currently only included in the "Experimental" channel of Gateway API. For more information on release channels, refer to our [versioning guide](https://gateway-api.sigs.k8s.io/concepts/versioning).
```